app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -104,7 +104,7 @@
 
             <com.google.android.material.textfield.TextInputEditText
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
+                android:layout_height="wrap_content"
                 android:inputType="textPassword" />
```

Comment:
Is this change related to this PR?

Reply:
No but the format was inconsistent. Other fields had the height as wrap content. Except confirm password field. So thought to change that.

Reply:
If this change doesn't belong to this PR, I suggest you remove it. Usually, for things like this, a new PR should be submitted.

Reply:
Ok